src/main/java/org/zalando/nakadi/repository/TopicRepository.java

```diff
@@ -77,8 +77,6 @@ void validateReadCursors(List<NakadiCursor> cursors) throws InvalidCursorExcepti
 
     long totalEventsInPartition(Timeline timeline, String partition);
 
-    long numberOfEventsBeforeCursor(NakadiCursor cursor);
```

Comment:
The proposed solution will work as a quick fix. Do you think we should file a ticket to refactor it later?

Reply:
@adyach absolutely, I even mention it in the PR description. I will create a Jira ticket for it.